src/librustdoc/html/layout.rs

```diff
@@ -70,6 +70,10 @@ r##"<!DOCTYPE html>
         {sidebar}
     </nav>
 
+    <div id="theme-picker">&#x1f58c;
```

Comment:
`theme-picker` and `theme-choices` need some ARIA attributes (see [this](https://github.com/rust-lang-nursery/mdBook/blob/master/src/theme/index.hbs#L78) for how it's handled in `mdBook`).

Reply:
In mdbook it breaks elements alignment so I'd prefer not use the same mechanisms.

Reply:
How does it break elements alignment? Also, this should be a button.

Reply:
I don't see the point of this being a button... :-/

Reply:
Can you click it? Then it should be a button or a link.